### PR TITLE
Excel js open spec

### PR DESCRIPTION
**Pull request by mingwan**
change some properties and comments in charting API

### CODE REVIEW

File: reference/excel/chartformatstring.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ChartFormatString Object (JavaScript API for Excel)
 
-Represents the characters in chart object that contains text, like chart title, chart axis title, etc.
+Represents the substring in chart related objects that contains text, like ChartTitle object, ChartAxisTitle object, etc.
```
Comment (Lauragra) [inline reference/excel/chartformatstring.md]: Please avoid using "etc." as that is ambiguous. You can change "like" to "for example" here and remove etc.

File: reference/excel/chartformatstring.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ None
 ## Relationships
 | Relationship | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|font|[ChartFont](chartfont.md)|Represents the font properties of chartFormatString object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|font|[ChartFont](chartfont.md)|Represents the font attributes, such as font name, font size, color, etc. of chart characters object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/chartformatstring.md]: Please remove the "etc." here and list all the font attributes, or make it clear that these are examples.

File: reference/excel/chartpointformat.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ None
 ## Relationships
 | Relationship | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|border|[ChartBorder](chartborder.md)|Represents the border format of a chart point, which includes border formating information. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|border|[ChartBorder](chartborder.md)|Represents the border format of a chart point, which includes border formating information. Read-only Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/chartpointformat.md]: Please remove the dupe Read-only.

File: reference/excel/chartseriescollection.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ None
 |[add(name: string, index: number)](#addname-string-index-number)|[ChartSeries](chartseries.md)|Add a new series to the collection.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
 |[getCount()](#getcount)|int|Returns the number of series in the collection.|[1.4](../requirement-sets/excel-api-requirement-sets.md)|
 |[getFirst()](#getfirst)|[ChartSeries](chartseries.md)|Gets the first series in the collection.|[1.7](../requirement-sets/excel-api-requirement-sets.md)|
-|[getItemAt(index: number)](#getitematindex-number)|[ChartSeries](chartseries.md)|Retrieves a series based on its position in the collection|[1.1](../requirement-sets/excel-api-requirement-sets.md)|
+|[getItemAt(index: number)](#getitematindex-number)|[ChartSeries](chartseries.md)|Retrieves a series based on its position in the collection.c|[1.1](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/chartseriescollection.md]: Is the ".c" a typo?

File: reference/excel/charttitle.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ Represents a chart title object of a chart.
 
 | Property	   | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|horizontalAlignment|string|Represents the horizontal alignment for chart title. Possible values are: Center, Left, Right, Justify, Distributed.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|horizontalAlignment|string|Represents the horizontal alignment for the specified objec Possible values are: Center, Left, Right, Justify, Distributed.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/charttitle.md]: Please fix the typo "object" - should be "object."

File: reference/excel/charttitle.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ _See property access [examples.](#property-access-examples)_
 
 | Method		   | Return Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|[getSubstring(start: number, length: number)](#getsubstringstart-number-start-number)|[ChartFormatString](chartformatstring.md)|Get the characters of a chart title. Line break '\n' also counts one charater.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|[getSubstring(start: number, start: number)](#getsubstringstart-number-start-number)|[ChartFormatString](chartformatstring.md)|Get the characters of a chart title. Line break '\n' also counts one charater.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/charttitle.md]: Is the second "start: number" a dupe?

File: reference/excel/chartaxis.md [hunk 7]
```diff
@@ -31,11 +30,29 @@ _See property access [examples.](#property-access-examples)_
...
+
+#### Syntax
+```js
+chartAxisObject.setCategoryNames(sourceData);
+```
+
+#### Parameters
+| Parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+|sourceData|Range or string()|The Range object corresponding to the source data, or string array|
```
Comment (Lauragra) [inline reference/excel/chartaxis.md]: Change "corresponding" to "that corresponds", and add a period here.

**Comment by mingwan** [comment]
Already updated based on feedbacks.

File: reference/excel/chartaxis.md [hunk 8]
```diff
@@ -31,11 +30,29 @@ _See property access [examples.](#property-access-examples)_
...
+
+#### Syntax
+```js
+chartAxisObject.setCategoryNames(sourceData);
+```
+
+#### Parameters
+| Parameter	   | Type	|Description|
+|:---------------|:--------|:----------|
+|sourceData|Range or string()|The Range object that corresponds to the source data, or string array|
```
Comment (Lauragra) [inline reference/excel/chartaxis.md]: Add a period after the Description.

File: reference/excel/chartformatstring.md [hunk 9]
```diff
@@ -9,7 +9,7 @@ None
 ## Relationships
 | Relationship | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|font|[ChartFont](chartfont.md)|Represents the font properties of chartFormatString object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|font|[ChartFont](chartfont.md)|Represents the font attributes of chart substring object, such as font name, font size, color . Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```
Comment (Lauragra) [inline reference/excel/chartformatstring.md]: Please add the article "the" before "chart substring. Also add "or" before "color".

File: reference/excel/charttrendlinecollection.md [hunk 10]
```diff
@@ -36,7 +35,7 @@ chartTrendlineCollectionObject.add(type);
 #### Parameters
 | Parameter	   | Type	|Description|
 |:---------------|:--------|:----------|
-|type|string|Optional. Specifies the trendline type.  Possible values are: Linear, Expontential, Logarithmic, MovingAvg, Polynomial, Power|
+|type|string|Optional. TrendlineType.  Possible values are: Linear, Expontential, Logarithmic, MovingAvg, Polynomial, Power|
```
Comment (Lauragra) [inline reference/excel/charttrendlinecollection.md]: Add a period at the end here too.

**Comment by Lauragra** [comment]
Looks good!

**Pull request merged**